releases/1.1.1-SNAPSHOT.yml

```diff
@@ -0,0 +1,3 @@
+version: 1.1.1-SNAPSHOT
+rpm_version: 1.1.1~SNAPSHOT
+deb_version: 1.1.1~SNAPSHOT
```

Comment:
You can't have dashes in the RPM version because dash is the separator between version and release.

Some versions of RHEL complain of the tilde IIRC. I've used underscore in the past like 1.1.1_SNAPSHOT, but then you have an upgrade problem because 1.1.1_SNAPSHOT is actually newer than 1.1.1 using the rpm version comparison algorithm. There is no happy middle ground. :disappointed: